src/Components/CreateImageWizard/steps/Registration/AAPRegistration.tsx

```diff
@@ -0,0 +1,161 @@
+  selectAapJobTemplateId,
+  selectAapHostConfigKey,
+  selectAapTlsCertificateAuthority,
+} from '../../../../store/wizardSlice';
+import { useRegistrationValidation } from '../../utilities/useValidation';
+import { ValidatedInputAndTextArea } from '../../ValidatedInput';
+
+const AAPRegistration = () => {
+  const dispatch = useAppDispatch();
+  const controllerUrl = useAppSelector(selectAapControllerUrl);
+  const jobTemplateId = useAppSelector(selectAapJobTemplateId);
+  const hostConfigKey = useAppSelector(selectAapHostConfigKey);
+  const tlsCertificateAuthority = useAppSelector(
+    selectAapTlsCertificateAuthority
+  );
+  const [isRejected, setIsRejected] = React.useState(false);
+  const stepValidation = useRegistrationValidation();
+
+  const validated = stepValidation.errors['certificate']
+    ? 'error'
+    : stepValidation.errors['certificate'] === undefined &&
+      tlsCertificateAuthority
+    ? 'success'
+    : 'default';
+
+  const handleControllerUrlChange = (value: string) => {
+    dispatch(changeAapControllerUrl(value));
+  };
+
+  const handleJobTemplateIdChange = (value: string) => {
+    dispatch(changeAapJobTemplateId(value));
+  };
+
+  const handleHostConfigKeyChange = (value: string) => {
+    dispatch(changeAapHostConfigKey(value));
+  };
+
+  const handleClear = () => {
+    dispatch(changeAapTlsCertificateAuthority(''));
+  };
+
+  const handleTextChange = (
+    _event: React.ChangeEvent<HTMLTextAreaElement>,
+    value: string
+  ) => {
+    dispatch(changeAapTlsCertificateAuthority(value));
+  };
+
+  const handleDataChange = (_: DropEvent, value: string) => {
+    dispatch(changeAapTlsCertificateAuthority(value));
+  };
+
+  const handleFileRejected = () => {
+    dispatch(changeAapTlsCertificateAuthority(''));
+    setIsRejected(true);
+  };
+
+  return (
+    <>
+      <FormGroup label="Ansible Controller URL" isRequired>
+        <ValidatedInputAndTextArea
+          value={controllerUrl || ''}
+          onChange={(_event, value) => handleControllerUrlChange(value)}
+          ariaLabel="ansible controller url"
+          isRequired
+          stepValidation={stepValidation}
+          fieldName="controllerUrl"
+          placeholder=""
+        />
+      </FormGroup>
+      <FormGroup label="Job Template ID" isRequired>
+        <ValidatedInputAndTextArea
+          value={jobTemplateId || ''}
+          onChange={(_event, value) => handleJobTemplateIdChange(value)}
+          ariaLabel="job template id"
+          isRequired
+          stepValidation={stepValidation}
+          fieldName="jobTemplateId"
+          placeholder=""
+        />
+      </FormGroup>
+      <FormGroup label="Host Config Key" isRequired>
+        <ValidatedInputAndTextArea
+          value={hostConfigKey || ''}
+          onChange={(_event, value) => handleHostConfigKeyChange(value)}
+          ariaLabel="host config key"
+          isRequired
+          stepValidation={stepValidation}
+          fieldName=""
```

Comment:
The validation wouldn't appear for this field.

```suggestion
          fieldName="hostConfigKey"
```